python/llm/example/GPU/HuggingFace/Multimodal/MiniCPM-V-2/generate.py

```diff
@@ -157,16 +157,14 @@ def _pos_embed(self, x: torch.Tensor) -> torch.Tensor:
     # here the prompt tuning refers to https://huggingface.co/openbmb/MiniCPM-V-2/blob/main/README.md
     msgs = [{'role': 'user', 'content': args.prompt}]
```

Comment:
It seems with ipex-llm >= 0815, MiniCPM-V-2 can also use fp16. Please check it and maybe update this example too.